### ISSUE TITLE
Support new headless Chrome --headless=new

### ISSUE THREAD

**Issue by alfonmga** [go-rod/rod#818]
The [new headless Chrome](https://antoinevastel.com/bot%20detection/2023/02/19/new-headless-chrome.html) has been released in November 2022. 
In this issue we could explore if Rod is stable for it.

Problems I encounter so far:
- [On MacOS the Chromium application is visible in the dock](https://user-images.githubusercontent.com/9363272/220989531-5d61446c-3c0e-4942-91a1-5a1c398a9ce3.png).

**Comment by Fly-Playgroud** [comment]
Yeah, I have also noticed this new feature of Chrome, but this new feature will cause new bugs, which will cause the problem that `--remote-debugging-address` will not take effect. It will only listen on `127.0.0.1`. For details : https://bugs.chromium.org/p/chromium/issues/detail?id=1373872&q=remote%20debugging%20address&can=2

**Comment by Fly-Playgroud** [comment]
> Rod Version: v0.112.6
> 
> The [new headless Chrome](https://antoinevastel.com/bot%20detection/2023/02/19/new-headless-chrome.html) has been released in November 2022. In this issue we could explore if Rod is stable for it.
> 
> ```go
> package main
> 
> import (
> 	"github.com/go-rod/rod"
> 	"github.com/go-rod/rod/lib/launcher"
> )
> 
> func main() {
> 	u := launcher.New().Set("headless", "new").MustLaunch() // use new headless mode
> 	page := rod.New().ControlURL(u).MustConnect().MustPage("https://www.wikipedia.org/")
> 	page.MustWaitLoad().MustScreenshot("a.png")
> }
> ```
> 
> Problems I have encountered so far:
> 
> * [On MacOS the Chromium application is visible in the dock](https://user-images.githubusercontent.com/9363272/220989531-5d61446c-3c0e-4942-91a1-5a1c398a9ce3.png).

Hi, here you can give a temporary support solution: if you want to use the `new` Headless Mode, please use the` Launcher.Set (flags.Headless, "new") `).
At present, `Launcher.Headless (enable bool)` this helper func will activate the `old` Headless Mode. Over time, this Helper Func will be activated by default to` New` Headless Mode. The length of this time depends on the update of Chrome. Of course, you can also display `Launcher.Set (flags.Headless," old ")`  `old` Headless Mode.

**Comment by alfonmga** [comment]
> this new feature will cause new bugs, which will cause the problem that `--remote-debugging-address` will not take effect. It will only listen on `127.0.0.1`. For details : https://bugs.chromium.org/p/chromium/issues/detail?id=1373872&q=remote%20debugging%20address&can=2

I didn't know about this bug, thanks for the heads up.

> Hi, here you can give a temporary support solution: if you want to use the new Headless Mode, please use theLauncher.Set (flags.Headless, "new")).

Unfortunately, it doesn't fix the issue I mentioned. The Chromium application is still visible in the dock.
It works fine on Windows, though.
